src/login/logind-user.c

```diff
@@ -344,6 +347,23 @@ int user_load(User *u) {
         return 0;
 }
 
+static void user_start_runtime_dir(User *u) {
```

Comment:
feels weird if user_start_service_manager() returns an erorr, bit user_start_runtime_dir() does not. I think both should return an error, to keep the symmetry, even if the return value is cast away in one case. it just reduces suprises